src/components/Text/docs/index.story.tsx

```diff
@@ -18,5 +34,42 @@ export default {
   exampleProps: {
     typography: Object.keys(TYPOGRAPHY).map(key => TYPOGRAPHY[key]),
   },
-  examples: <Examples />,
+  sections: [
+    header(),
+    tabs([
+      tab({
+        title: 'Usage',
+        sections: [
+          description(
+            '`Text` is a component allowing to render a custom text.',
+          ),
+
+          importExample({
+            source: examples.importExample,
+          }),
+
+          divider(),
+
+          title('Examples'),
+
+          ...[
+            {
+              title: 'Basic Usage',
+              source: examples.basic,
+            },
+          ].map(code),
+        ],
+      }),
+
+      ...[
+        { title: 'Playground', sections: [playground()] },
+        { title: 'API', sections: [api()] },
+        {
+          title: 'Style API',
+          sections: [settingsApi()],
+        },
+        { title: 'TestKit', sections: [testkit()] },
```

Comment:
I think we're missing a `Settings Panel` section here.
It should use the same variables as in https://github.com/wix/wix-ui-tpa/pull/498/files#diff-0d1834f28d3f1b4b44e8f2aca65636ce76a3c3770840f35da1f1be739aa36325L27